sections/generic-best-practices/await-dont-sleep.md

```diff
@@ -206,10 +206,9 @@ Cypress
 - waiting for an XHR request/response
 
 ```javascript
-cy.server()
-cy.route('http://dummy.restapiexample.com/api/v1/employees').as('employees')
+cy.intercept('http://dummy.restapiexample.com/api/v1/employees').as('employees')
 cy.wait('@employees')
```

Comment:
there is exactly a bug here with cy.intercept() as of 2 weeks ago; cy.wait() ing on an alias was not working. Let's leave it as this and wait for the team to address it

Reply:
As far as you know: did they fix the bug?

Reply:
Yes